.github/workflows/build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           ~/go/pkg/mod
           ${{ github.workspace }}/hack/tools/bin
           ${{ github.workspace }}/hack/tools/lib
-        key: ${{ runner.os }}-c-${{ hashFiles('hack/make-rules/tools.mk') }}-go-${{ hashFiles('**/go.sum') }}
+        key: ${{ runner.os }}-c-${{ hashFiles('hack/make-rules/tools.mk') }}-${{ hashFiles('hack/tools/requirements.sh') }}-go-${{ hashFiles('**/go.sum') }}
```

Comment:
the key was defined 9 times in the file, can we replace this long line by a single env variable ?

Reply:
I think not, please see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for information where `hashFiles` and `runner.os` can be used in the workflow.

Reply:
You are right.